server/src/database/setup.ts

```diff
@@ -45,8 +45,8 @@ const populateDb = () => {
         courses.forEach((course: any) => {
             const { courseTitle, courseCode, description, credits, preReqText, coReqText, preReqs = [], coReqs = [], sections = [] } = course;
             const [ courseDept, courseNumber ] = courseCode.split(" ");
-            coReqs.forEach((coReq: any) => handleReq(courseDept, courseNumber, coReq, coReqText, coReqsToStore));
-            preReqs.forEach((preReq: any) => handleReq(courseDept, courseNumber, preReq, preReqText, preReqsToStore));
+            // coReqs.forEach((coReq: any) => handleReq(courseDept, courseNumber, coReq, coReqText, coReqsToStore));
+            // preReqs.forEach((preReq: any) => handleReq(courseDept, courseNumber, preReq, preReqText, preReqsToStore));
```

Comment:
I think we should remove/comment out L27-28 where we create the CoReq and PreReq tables and any other instance where we reference or operate on those 2 tables. There's a few in the try block below.

Reply:
And we could just delete the related lines of code for now since we would have it in the history! We just need to remind ourselves to revisit this pre-req/co-req issue later!